src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/meta_helper_card.html.twig

```diff
@@ -0,0 +1,43 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain "Admin.Shopparameters.Feature" %}
+
+{% block meta_helper_card %}
+  <div id="seo-urls-helper-block" class="empty-state card">
+    <div class="empty-state__left shape-one helper-card__meta-background">
+      <img src="{{ asset('themes/new-theme/scss/img/helper-card/seo@3x.png') }}">
```

Comment:
I'm not sure the image should be stored inside the scss folder

Reply:
I agree but I used the same location as other images were stored - the best case scenario would be that these images should generate itself in the public directory.
Should I keep them there or move elsewhere for now. Maybe in themes/new-theme/img folder for instance?

Reply:
I'd like to see them in the root `img`folder if possible, don't forget to update the other showcase block done by @CaptainYouz :+1: